src/test/java/com/brandwatch/kafka_pod_autoscaler/triggers/kafka/LagModelTest.java

```diff
@@ -0,0 +1,200 @@
+        return List.of(call(2, evenOffsets(0L), evenOffsets(0L)));
+    }
+
+    private static List<UpdateCallParameters> unmovingTopic() {
+        return List.of(
+            call(2, evenOffsets(0L), evenOffsets(0L)),
+            call(2, evenOffsets(0L), evenOffsets(0L)),
+            call(2, evenOffsets(0L), evenOffsets(0L))
+        );
+    }
+
+    private static List<UpdateCallParameters> stuckConsumer() {
+        return List.of(
+            call(2, evenOffsets(0L), evenOffsets(0L)),
+            call(2, evenOffsets(0L), evenOffsets(1L)),
+            call(2, evenOffsets(0L), evenOffsets(2L))
+        );
+    }
+
+    private static List<UpdateCallParameters> slowConsumer() {
+        return List.of(
+            call(2, evenOffsets(0L), evenOffsets(1L)),
+            call(2, evenOffsets(1L), evenOffsets(4L)),
+            call(2, evenOffsets(2L), evenOffsets(8L))
+        );
+    }
+
+    private static List<UpdateCallParameters> keepingUpConsumer() {
+        return List.of(
+            call(2, evenOffsets(1L), evenOffsets(1L)),
+            call(2, evenOffsets(4L), evenOffsets(4L)),
+            call(2, evenOffsets(8L), evenOffsets(8L))
+        );
+    }
+
+    private static List<UpdateCallParameters> catchingUpConsumer() {
+        return List.of(
+            call(2, evenOffsets(1L), evenOffsets(4L)),
+            call(2, evenOffsets(3L), evenOffsets(5L)),
+            call(2, evenOffsets(6L), evenOffsets(6L))
+        );
+    }
+
+    private static List<UpdateCallParameters> scalingCatchingUpConsumer() {
+        return List.of(
+            call(2, evenOffsets(0L), evenOffsets(2L)),
+            call(2, evenOffsets(1L), evenOffsets(4L)),
+            call(4, evenOffsets(4L), evenOffsets(6L)),
+            call(4, evenOffsets(7L), evenOffsets(8L)),
+            call(4, evenOffsets(9L), evenOffsets(10L)),
+            call(4, evenOffsets(12L), evenOffsets(12L))
+        );
+    }
+
+    private static Map<TopicPartition, Long> evenOffsets(long offset) {
+        return Map.of(
+            PARTITION_1, offset,
+            PARTITION_2, offset,
+            PARTITION_3, offset,
+            PARTITION_4, offset
+        );
+    }
+
+    private static UpdateCallParameters call(int replicaCount,
+                                             Map<TopicPartition, Long> consumerOffsets,
+                                             Map<TopicPartition, Long> topicEndOffsets) {
+        return new UpdateCallParameters(replicaCount, consumerOffsets, topicEndOffsets);
+    }
+
+    private static ExpectedResults expect(int replicaCount, long lag, OptionalDouble consumerRate, OptionalDouble topicRate) {
+        return new ExpectedResults(replicaCount, lag, consumerRate, topicRate);
+    }
+//
+//    @ParameterizedTest
+//    @MethodSource
+//    public void estimateLoadedConsumerRate(int previousReplicaCount,
+//                                      List<Map<TopicPartition, Long>> previousConsumerOffsets,
```

Comment:
are these tests supposed to be commented out?

Reply:
Ah, meant to delete these 🤦